Readme.md

```diff
@@ -0,0 +1,147 @@
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 MIRANJA TRACY <a name="about-project"></a>
+
+**MIRANJA TRACY hello world** is a webpage That i have created to test whether i grasp steps i learnt.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/es/docs/Web/HTML">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/es/docs/Web/CSS">CSS</a></li>
+  </ul>
+</details>
+ 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+ 
+<!-- LIVE DEMO -->
+ 
+## 🚀 Live Demo <a name="live-demo"></a>
+ 
+- [Live Demo Link](https://github.com/Tracy-miranja/Myfirst-repo)
+ 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+ 
+<!-- GETTING STARTED -->
+ 
+## 💻 Getting Started <a name="getting-started"></a>
+ 
+To get a local copy up and running, follow these steps.
+ 
+### Prerequisites
+ 
+In order to run this project you need:
+ 
+- A computer.
+- Internet access.
+- A modern web browser.
+ 
+### Setup
+ 
+Clone this repository to your desired folder:
+ 
+```sh
+  cd my-folder
+  git clone git@github.com:Tracy-miranja/Tracy-miranja/hello-microverse.git
+```
+ 
+### Install
+ 
+Install this project with:
+ 
+```sh
+  cd my-folder
+  git clone git@github.com:Tracy-miranja/Tracy-miranja/hello-microverse.git
+```
+### Usage
+ 
+To run the project, open index.html in your web browser.
+ 
+<!-- AUTHORS -->
+ 
+## 👥 Author <a name="author"></a>
+ 
+ 
+👤 **Tracy miranja**
+ 
+- GitHub: [@githubhandle](https://github.com/Tracy-miranja)
+- Twitter: [@twitterhandle](https://twitter.com/tracymiranja)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/tracymiranja)
+ 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+ 
+<!-- FUTURE FEATURES -->
+ 
+## 🔭 Future Features <a name="future-features"></a>
+ 
+ 
+- [ ] **Web page body**
+- [ ] **Web page footer**
+- [ ] **Desktop design**
+ 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+ 
+<!-- CONTRIBUTING -->
+ 
+## 🤝 Contributing <a name="contributing"></a>
+ 
+Contributions, issues, and feature requests are welcome!
+ 
+Feel free to check the [issues page](../../issues/).
+ 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+ 
+<!-- SUPPORT -->
+ 
+## ⭐️ Show your support <a name="support"></a>
+ 
+If the project was great as your view, don't hesitate to share.
+ 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+ 
+<!-- ACKNOWLEDGEMENTS -->
+ 
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+ 
+I would like to thank to all my Mricronauts partners...
+ 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+ 
+<!-- LICENSE -->
+ 
+## 📝 License <a name="license"></a>
+ 
+This project is [MIT](./LICENSE) licensed.
+ 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+README.md
+Displaying README.md.
```

Comment:
- [ OPTIONAL] Whan I click on the MIT link, I am redirected to an error page. Kindly add a valid LICENSE by following the commands in the article below:

 https://choosealicense.com/licenses/mit/


#### You can pay attention to this part of the article in order to implement it properly. 👍 

![image](https://user-images.githubusercontent.com/77106962/211669641-c93eda24-0663-44d8-b079-4bbff53d37fe.png)